mono/tests/Makefile.am

```diff
@@ -1,24 +1,40 @@
 SUBDIRS = assemblyresolve gc-descriptors
 
-if INSTALL_AOT_ONLY
-FEATUREFUL_RUNTIME_TEST =  
-else
-FEATUREFUL_RUNTIME_TEST = test-appdomain-unload
-endif
+check-local:
+	$(MAKE) test-runtime
+	$(MAKE) test_platform
```

Comment:
Oh I see this was already there before. might still be worth to make it consistent.